go.mod

```diff
@@ -22,4 +22,5 @@ require (
 replace (
 	github.com/v3io/frames => ./
 	github.com/xwb1989/sqlparser => github.com/v3io/sqlparser v0.0.0-20190306105200-4d7273501871
+	github.com/v3io/v3io-go => github.com/v3io/v3io-go v0.2.4-0.20201209122348-e01fabe35df8
```

Comment:
Hm, there is no v0.2.4. Should be v0.0.0 like the line above it. Where is e01fabe35df8 though? I don't see such a commit at all.

Reply:
https://github.com/v3io/v3io-go/commit/e01fabe35df8ef68f13fed4eb0caa22e190ffd3b. 
this is the commit. the v.0.2.4 is build automatically when the hash is added